README.md

```diff
@@ -193,16 +193,11 @@ contentSecurityPolicy: {
 In order to use the addon, you must first [configure](#configuration) it, then inject it into any Object registered in the container that you wish to track. For example, you can call a `trackPage` event across all your analytics services whenever you transition into a route, like so:
 
 ```js
-// app/router.js
-import EmberRouter from '@ember/routing/router';
-import config from './config/environment';
-import { get } from '@ember/object';
+// app/routes/application.js
+import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
-import { scheduleOnce } from '@ember/runloop';
-
-const Router = EmberRouter.extend({
-  location: config.locationType,
 
+export default Route.extend({
```

Comment:
@locks are you sure that the `this.on('routeDidChange'` still works correctly after changing this from Router to Route?

Reply:
You are correct, it should have been `this.router.on`.
Alternatively it can remain on `app/router.js`, and we can avoid the endless loop by not injecting the router service. I'll do a follow up.

Reply:
@locks any news on this?

Reply:
still in my queue 😅